jekyll/_cci2/authentication.md

```diff
@@ -0,0 +1,40 @@
+---
+ layout: classic-docs
+ title: "Authentication"
+ short-title: "Authentication"
+ description: "Configuring LDAP Authentication"
+ categories: [administration]
+ hide: true
+ order: 9
+---
+
+This document describes how to enable, configure, and test CircleCI to authenticate users with OpenLDAP or Active Directory credentials.
+
+## Prerequisites
+
+- Install and configure an LDAP instance and populate the database with your users and credentials.
+- Install a new instance of CircleCI 2.0 with no existing users using the [Installing CircleCI 2.0 on Amazon Web Services with Terraform]({{ site.baseurl }}/2.0/aws/) document.
```

Comment:
I would recommend highlighting the fact that LDAP should not be used for existing installations. Also, we need a disclaimer that this feature is experimental and in alpha stage.

Reply:
Thanks, I added: "**Note:** LDAP is not supported with existing installations, only clean installations may use LDAP. The LDAP authentication feature is currently only a preview."